src/Arduboy.cpp

```diff
@@ -26,7 +27,32 @@ void Arduboy::start() // deprecated
 void Arduboy::begin()
 {
   boot(); // required
+  bootUtils();
 
+  bootLogo();
+
+  // Audio
+  tunes.initChannel(PIN_SPEAKER_1);
```

Comment:
Yeah, set

//#define ARDUBOY_10   //< compile for the production Arduboy v1.0
#define AB_DEVKIT    //< compile for the official dev kit
#endif